### DISCUSSION TITLE
Welcome to xgplayer Discussions!

### DISCUSSION THREAD

**Discussion by gemxx** [bytedance/xgplayer#816] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LinyuNB** [comment]


**Reply by gemxx** [reply]
无法播放的问题不要再讨论区留言，可以新开一个issue并提供一下录屏文件

**Comment by cokestewpotato** [comment]
官方文档：setConfig更新配置信息，如果新的配置列表中包含内置插件的配置项，则会调用内置插件的setConfig对其进行更新
问题1：playnext插件没有setConfig方法，怎么动态更新下一集视频集合，目前代码使用的是player.getPlugin('playNext').config.urlList = xxArray
问题2：手动点击视频列表，通过player.playNext({url: '另一个url', poster: '新的海报图'， ... })切换视频，怎么更新下一集的idx，如不更新，点击下一集按钮的时候播放的视频信息不正确，目前使用的方式是player.getPlugin('playNext').idx = --index;

请帮确认以上使用方式是不是正确？或者有其他更好的api方式可以支持
